manifests/zone.pp

```diff
@@ -33,6 +36,7 @@
 # @param expire
 # @param negttl
 # @param serial
+# @param records
```

Comment:
Could you add a description here? We haven't done a great job in the past, but by documenting at least what we add we keep it in reasonable shape.

Reply:
Sorry for bothering you with this PR, I pinged you because you commented in the issue ^^.

I added a description in 487857bc6349ffc64d8bd9cced429a1f06c13942